client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js

```diff
@@ -5,6 +5,16 @@ import React from 'react';
 import { useSelector } from 'react-redux';
 import styled from '@emotion/styled';
 import { keyframes } from '@emotion/core';
+import {
+	isPlan,
+	isMonthly,
+	getYearlyPlanByMonthly,
+	getPlan,
+	isFreePlanProduct,
+	TERM_MONTHLY,
+	TERM_ANNUALLY,
+	TERM_BIENNIALLY,
+} from '@automattic/calypso-products';
```

Comment:
Nit: It would be nice to have this import be in alphabetical order.
```suggestion
import {
	getPlan,
	getYearlyPlanByMonthly,
	isFreePlanProduct,
	isMonthly,
	isPlan,
	TERM_ANNUALLY,
	TERM_BIENNIALLY,
	TERM_MONTHLY,
} from '@automattic/calypso-products';
```